src/js/themes/base.js

```diff
@@ -639,8 +639,19 @@ export const generate = (baseSpacing = 24, scale = 6) => {
           dark: '#d9d9d9',
           light: '#d9d9d9',
         },
+        container: {
```

Comment:
just a callout that i'm not really sure what's supposed to be / not be in this file - i added what I felt was needed to prevent nullrefs in the code changes, but i'm also not sure if i should be using `?` operators more often in the code itself to prevent nullrefs